File: CocoaPods-static/Example/ViewController.m
//
//  ViewController.m
//  Example
//
//  Created by Victor Ilyukevich on 4/22/16.
//  Copyright © 2016 Open Source. All rights reserved.
//

#import "ViewController.h"
#import <FormatterKit/TTTTimeIntervalFormatter.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  TTTTimeIntervalFormatter *formatter = [[TTTTimeIntervalFormatter alloc] init];
  formatter.usesAbbreviatedCalendarUnits = YES;

  NSLog(@"%@ == 3 wk ago", [formatter stringForTimeInterval:-3600*24*7*3]);

}

@end

File: Carthage-OSX/Example/ViewController.swift
//
//  ViewController.swift
//  Example
//
//  Created by Victor Ilyukevich on 5/7/16.
//  Copyright © 2016 Open Source. All rights reserved.
//

import Cocoa
import FormatterKit

class ViewController: NSViewController {

  override func viewDidLoad() {
    super.viewDidLoad()

    let locationFormatter = TTTLocationFormatter()
    print(locationFormatter)

    let intervalFormatter = TTTTimeIntervalFormatter()
    intervalFormatter.usesAbbreviatedCalendarUnits = true
    print("\(intervalFormatter.stringForTimeInterval(-3600*24*7*3)) == 3 wk ago")
  }

}

